content/issues/4/unnatural-language/index.md

```diff
@@ -134,13 +134,13 @@ Another problem is the graphic variation common in early forms of Chinese, espec
             >。
           </p>
           <p class="translation">
-            The <cite>Documents</cite> state: ‘Gaozong
```

Comment:
@thatbudakguy for some reason, this cite tag introduces a line break on the page between "The" and "Documents". Also it seems strange to have the second line break when the text is phrased as a sentence: "The / documents / state..."

<img width="559" alt="Screenshot 2023-09-07 at 9 34 45 AM" src="https://github.com/Princeton-CDH/startwords/assets/5580840/a7c65090-83c3-4c2f-bf48-a537b65c62bb">

Reply:
Strange, I was serving your branch locally and that's what I got. But I just did a hard refresh and the line breaks are gone. Sorry for the confusion